aces/Loris
# [Notification/Permissions] Fix raisinbread data rels

## Brief summary of changes

Fixes data that fixes https://github.com/aces/Loris/issues/5457

#### Testing instructions (if applicable)

1. Doc repo test plan:
- Check that preference “Receive Document Repository Notifications” works.
- Check that if you have the 'Receive document repository notification' activated you will receive an email each time one of the following event occurs:
Addition, deletion or modification of a file
Addition of a category. Also check that the www address contained in the notification email is correct.

#### Links to related tickets (GitHub, Redmine, ...)

https://github.com/aces/Loris/issues/5457
